src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -160,6 +161,13 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('fileupload_storage_connectionstring',
                    options_list=['--fileupload-storage-connectionstring', '--fcs'],
                    help='The connection string for the Azure Storage account to which files are uploaded.')
+        c.argument('fileupload_storage_authentication_type',
+                   options_list=['--fileupload-storage-auth-type', '--fsa'],
+                   help='The authentication type for the Azure Storage account to which files are uploaded.'
+                        'Possible values are ''keyBased'' and ''identityBased''')
```

Comment:
Double single quote?

Reply:
This was a leftover from an earlier attempt at including single quotes inside the string, which has since been removed (hopefully) from all the other identity/key-based commands. Thanks for catching that!